geometry_msgs/Point goal # the goal to navigate to, in the map frame
---
